### PR TITLE
livy: add deployment

### LINKED ISSUE #75: Add Livy
Integrate Apache Livy to enable interacting with the cluster interactively, e.g. with Jupyter notebooks.

### PR DESCRIPTION
**Pull request by Nuttymoon**
#### Which issue(s) this PR fixes:

Fixes #75 

#### Additional comments:

This PR depends on https://github.com/TOSIT-IO/tdp-collection-extras/pull/16

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +45,9 @@ ansible-playbook deploy-all.yml
 
 **Note:** All the WebUIs are Kerberized, you need to have a working Kerberos client on your host, configure the KDC in your `/etc/krb5.conf` file and obtain a valid ticket. You can also access the WebUIs through [Knox](#knox).
 
-## Customised deployment
+## Customised Deployment
 
-Each of the below sections includes a high-level explanation of each possible step of the deployment using this repository.
+Each of the below sections includes a high-level explanation of each possible step of TDP deployment.
```
Comment (DanielJohnHarty) [inline README.md:50]: Each of the below sections includes a high-level explanation of each possible step of a TDP deployment.

**Review by DanielJohnHarty — COMMENTED** [review]
If we add this to getting-started master, well need the associated commit id for tdp-collection-extras which includes the livy role.

File: README.md (hunk 1 above)
Comment (Nuttymoon) [reply README.md:50]: Yes but I think the commit hash will change upon merge as the PR is squashed. So we have to wait until https://github.com/TOSIT-IO/tdp-collection-extras/pull/16 is merged.

File: README.md [hunk 2]
```diff
@@ -313,7 +323,34 @@ You can then access the WebUIs of the TDP services through Knox:
...
+# Create a session
+curl -k -u : --negotiate -X POST https://edge-01.tdp:8998/sessions \
+  -d '{"kind": "pyspark"}' -H 'Content-Type: application/json'
+# Get the session status (wait until it is "idle")
+curl -k -u : --negotiate -X GET https://edge-01.tdp:8998/sessions
+# Submit a snippet of code to the session
+curl -k -u : --negotiate -X POST https://edge-01.tdp:8998/sessions/1/statements \
+  -d '{"code": "1 + 1"}' -H 'Content-Type: application/json'
+# Get the statement result
+curl -sk -u : --negotiate -X GET https://edge-01.tdp:8998/sessions/1/statements/0
```
Comment (leopaul36) [inline README.md:350]: I think it should be `/sessions/0` on first launch

Comment (Nuttymoon) [reply README.md:350]: Yes probably

File: scripts/tdp-release-uris.txt [hunk 3]
```diff
@@ -36,4 +36,5 @@ https://github.com/TOSIT-FR/tez/releases/download/tez-0.9.1-TDP-0.1.0-SNAPSHOT/t
 https://dlcdn.apache.org/lucene/solr/7.7.3/solr-7.7.3.tgz
 # zookeeper
 https://archive.apache.org/dist/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz
-
+# livy
```
Comment (leopaul36) [inline scripts/tdp-release-uris.txt:39]: We're removing these lines as part of https://github.com/TOSIT-IO/tdp-getting-started/pull/74.

We should also do it here

**Review by leopaul36 — APPROVED** [review]

**Review by leopaul36 — APPROVED** [review]

**Pull request merged**